pytorch_lightning/profiler/pytorch.py

```diff
@@ -107,6 +107,15 @@ def __init__(self, schedule: Callable) -> None:
         if not _KINETO_AVAILABLE:
             raise ModuleNotFoundError("You are trying to use `ScheduleWrapper` which require kineto install.")
         self._schedule = schedule
+        self.reset()
```

Comment:
this is the only diff here, right?
the rest for `setup` and `pre_step` seems the same